ruff.toml

```diff
@@ -36,9 +36,16 @@ indent-width = 4
 target-version = "py312"
 
 [lint]
-# Currently only enabled for F (Pyflakes), I (isort), E,W (pycodestyle:Error/Warning), PLC/PLE (Pylint:Convention/Error) 
-# and PERF (Perflint) rules: https://docs.astral.sh/ruff/rules/
-select = ["F", "I", "E", "W", "PLC", "PLE", "PERF"]
+select = [
+    "ASYNC", # flake8-async - async checks
+    "E", # pycodestyle - error rules
+    "F", # pyflakes - general Python errors, undefined names
+    "I", # isort - import sorting
+    "PERF", # perflint - https://docs.astral.sh/ruff/rules/
```

Comment:
nit: maybe this should be "performance anti-pattern rules" and the rules list should be above

Reply:
oh woops, I misread what the original comment was trying to convey haha. Fixed!